packages/common/mui/src/lib/product-dashboard/components/dashboard-card.tsx

```diff
@@ -123,15 +123,19 @@ export const DashboardCard = ({
                 }}
```

Comment:
The issue here is that you need to check if there are NOTE incentives on the card or not. Some currencies no longer have NOTE incentives and therefore we should not show the NOTE icon.